src/rt-app.c

```diff
@@ -666,26 +659,64 @@ void *thread_body(void *arg)
 		case deadline:
 			fprintf(data->log_handler, "# Policy : SCHED_DEADLINE\n");
 			tid = gettid();
-			attr.size = sizeof(attr);
-			attr.sched_flags = 0;
-			attr.sched_policy = SCHED_DEADLINE;
-			attr.sched_priority = 0;
-			attr.sched_runtime = data->runtime;
-			attr.sched_deadline = data->deadline; 
-			attr.sched_period = data->period;
+			dl_params.size = sizeof(struct sched_attr);
+			dl_params.sched_flags = 0;
+			dl_params.sched_policy = SCHED_DEADLINE;
+			dl_params.sched_priority = 0;
+			dl_params.sched_runtime = sched_data->runtime;
+			dl_params.sched_deadline = sched_data->deadline; 
+			dl_params.sched_period = sched_data->period;
 
 			log_notice("[%d] period: %lu, exec: %lu, deadline: %lu",
-				data->ind, data->period, data->runtime, data->deadline);
-			break;
+				data->ind, sched_data->period, sched_data->runtime, sched_data->deadline);
```

Comment:
We don't print anything for the other classes, maybe we can remove this or we could change this to log_debug and add the same for the other cases as well?